### PR TITLE
SDI-443 visit room mapping wip

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: integration_tests/e2e/visitsVisitRoomMappings.cy.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+context('Visit Room Mappings', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubAuthUser')
+  })
+
+  context('With VISIT MAPPINGS role', () => {
+    beforeEach(() => {
+      cy.task('stubSignIn', ['ROLE_TEST_ROLE'])
+      cy.signIn()
```
Comment (lbennett-moj) [inline integration_tests/e2e/visitsVisitRoomMappings.cy.ts:14]: using as temporary feature switch

File: server/config.ts [hunk 2]
```diff
@@ -74,6 +74,14 @@ export default {
       },
       agent: new AgentConfig(Number(get('NOMIS_PRISONER_TIMEOUT_RESPONSE', 60000))),
     },
+    mapping: {
+      url: get('MAPPING_API_URL', 'http://localhost:8103', requiredInProduction) as string,
+      timeout: {
```
Comment (andymarke) [inline server/config.ts:79]: I think you might need this in the helm files else won't it stop the app from starting in dev?

**Review by andymarke — APPROVED** [review]
LGTM

File: server/views/pages/visits/viewRoomMappings.njk [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set pageTitle = applicationName + " - Room mappings" %}
+
+{% block content %}
+{{ govukBackLink({
+text: "Back",
+href: '/'
```
[the remark this answers is not in the document]
Comment (lbennett-moj) [reply server/views/pages/visits/viewRoomMappings.njk:14]: yes -  wip

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**